modular_skyrat/master_files/code/modules/pain_shock/status_indicator.dm

```diff
@@ -0,0 +1,198 @@
+#define PLANE_STATUS -12 //Status Indicators that show over mobs' heads when certain things like stuns affect them.
+
+#define VIS_STATUS			24
+
+#define VIS_COUNT			24 //Must be highest number from above.
+#define STATUS_LAYER -2.1
+#define STATUS_INDICATOR_Y_OFFSET 2 // Offset from the edge of the icon sprite, so 32 pixels plus whatever number is here.
+#define STATUS_INDICATOR_ICON_X_SIZE 0 // Don't need to care about the Y size due to the origin being on the bottom side.
+#define STATUS_INDICATOR_ICON_MARGIN 0 // The space between two status indicators. We don't do this with the current icons.
+
+/datum/preference/toggle/enable_status_indicators
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "enable_status_indicators"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/datum/preference/toggle/enable_status_indicators/create_default_value()
+	return TRUE
+
+/datum/preference/toggle/enable_status_indicators/apply_to_client(client/client, value)
+	. = ..()
+	var/atom/movable/screen/plane_master/runechat/status/status = locate() in client?.screen
+	if(!status)
+		return
+	if(value)
+		status.alpha = 255
+	else
+		status.alpha = 0
+/mob
+	var/status_enabled = TRUE
+/mob/living
+	var/list/status_indicators = null // Will become a list as needed.
```

Comment:
I'd also like for you to document these variables, since you're adding them. For instance, I'm not sure what kind of list `status_indicator` is supposed to be, if it's supposed to be a list of strings or a list of images.

Also, what even is the point of `status_enabled` here? I haven't seen it come up anywhere else in your code so far.
```suggestion
/mob
	var/status_enabled = TRUE

/mob/living
	var/list/status_indicators = null // Will become a list as needed.
```

Reply:
status_enabled gone next commit